assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/datesListTableRow.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { format } from 'date-fns';
+import { __ } from '@wordpress/i18n';
+
+import DateRegistrationsLink from '@edtrUI/datetimes/DateRegistrationsLink';
+import DateActionsMenu from '@edtrUI/datetimes/datesList/actionsMenu/DateActionsMenu';
+import { Datetime } from '../../../../services/apollo/types';
+import { DisplayDates } from '../../../../interfaces/datetimes/types';
+import { getBackgroundColorClassName, status } from '@sharedEntities/datetimes/helpers';
+import { InlineEditText } from '@appInputs/InlineEditInput';
+import { shortenGuid } from '@appServices/utilities/text';
+import { useDatetimeMutator } from '@edtrServices/apollo/mutations';
+
+import '../../../../../../application/ui/styles/root/entity-status.css';
+
+const DATE_TIME_FORMAT = 'EEE MMM yy h:mm a';
```

Comment:
maybe move this to `\application\constants\dateFnsFormats.ts` and name accordingly (`ENTITY_LIST_DATE_TIME_FORMAT`?!?)

**OR**...

get it from the `dateTimeFormats` in `useConfigData()` so that the list simply displays whatever format the user has set in the WP admin

Reply:
went with the first proposal
***
at some point in the future it will be nice to go with the second option, but instead of pulling the config inside of `datesListTableRow` it would be more maintainable to have a component that glues together the config and the date value, such that `datesListTableRow` doesn't have to deal with any config logic, if this make sense

Reply:
https://github.com/eventespresso/event-espresso-core/pull/2500/commits/cbc30a9de25f556124f7977b5eaa7ae71585c610